MAINTAINERS.md

```diff
@@ -6,59 +6,61 @@
+| Alex Eftimie              | [alexef](https://github.com/alexef)                     | Reviewer - CD                                                                | [GetYourGuide](https://www.getyourguide.com/)        |
+| Jann Fischer              | [jannfis](https://github.com/jannfis)                   | Approver - CD                                                                | [Red Hat](https://www.github.com/redhat/)            |
+| Dan Garfield              | [todaywasawesome](https://github.com/todaywasawesome)   | Approver(docs) - CD                                                          | [Codefresh](https://www.github.com/codefresh/)       |
+| Alexandre Gaudreault      | [agaudreault](https://github.com/agaudreault)           | Approver - CD, Reviewer - Rollouts                                           | [Intuit](https://www.github.com/intuit/)             |
+| Christian Hernandez       | [christianh814](https://github.com/christianh814)       | Reviewer(docs) - CD                                                          | [Akuity](https://akuity.io/)                         |
+| jswxstw                   | [jswxstw](https://github.com/jswxstw)                   | Reviewer - Workflows                                                         | Independent                                          |
+| Saumeya Katyal            | [saumeya](https://github.com/saumeya)                   | Reviewer - CD                                                                | [Red Hat](https://www.github.com/redhat/)            |
+| Kostis Kapelonis          | [kostis-codefresh](https://github.com/kostis-codefresh) | Reviewer - Rollouts                                                          | [Codefresh](https://www.github.com/codefresh/)       |
+| Pasha Kostohrys           | [pasha-codefresh](https://github.com/pasha-codefresh)   | Approver - CD                                                                | [Codefresh](https://www.github.com/codefresh/)       |
+| Nitish Kumar	             | [nitishify](https://github.com/nitishify)               | Reviewer - CD                                                                | [Akuity](https://akuity.io/)                         |
+| Ed Lee                    | [edlee2121](https://github.com/edlee2121)               | Approver - Workflows, Events                                                 | [Intuit](https://www.github.com/intuit/)             |
+| Vlad Losev                | [vladlosev](https://github.com/vladlosev)               | Approver(helm-chart) - CD                                                    | [Sage Intacct](https://github.com/intacct)           |
+| Mason Malone              | [MasonM](https://github.com/MasonM)                     | Reviewer - Workflows                                                         |                                                      |
```

Comment:
FYI: I work for Adobe, so might want to mention that in the "Affiliation" column
```suggestion
| Mason Malone              | [MasonM](https://github.com/MasonM)                     | Reviewer - Workflows                                                         |  [Adobe](https://www.adobe.com/)                                                    |
```

Reply:
Added!